src/main/java/core/basesyntax/FootballMatch.java

```diff
@@ -15,6 +14,13 @@
      * если не угадал ничего - возвращать 0.</p>
      */
     public int matchResult(int result1, int result2, int bet1, int bet2) {
+        if (result1 == bet1 && result2 == bet2) {
```

Comment:
use ternary operator instead of if/else. see this [checklist](https://mate-academy.github.io/jv-program-fulltime/01_core/checklist/hw_01.html) before sending link